app/src/main/java/com/realityexpander/tasky/agenda_feature/presentation/event_screen/EventEditors.kt

```diff
@@ -129,19 +139,22 @@ fun EventPropertyEditors(
                 buttons = {
                     positiveButton(
                         text = stringResource(R.string.ok),
-                        disableDismiss = true,
+                        disableDismiss = true,  // don't close the dialog automatically when OK button is tapped
```

Comment:
Why not?

Reply:
Because if there is an Error, i want the dialog to stay open. Without this flag, the OK button automatically closes the dialog, and I want it to stay open until the attendee is verified as existing and added.

Reply:
I will remove this at it seems to have no effect.